### ISSUE TITLE
Compatible with Keras?

### ISSUE THREAD

**Issue by andreaskeviin** [facebookresearch/madgrad#6]
I'm wondering if this is compatible with Keras 

Also, I'm a bit confused on using this optimizer, is it just a drop in replacement for adam?
for example, 

```
import madgrad

opt = Madgrad(lr=0.001)
```

**Comment by adefazio** [comment]
Currently there is no Tensorflow/Keras implementation, but pull requests are welcome. Yes it is a drop-in replacement for Adam, although the learning rate and weight decay will need to be changed, it won't work well at Adam defaults.
